src/locales/en.js

```diff
@@ -1,15 +1,15 @@
 export default {
   'login.title': 'LOGIN',
-  'login.signup': 'Sign up',
+  'login.signup': 'SIGN UP',
```

Comment:
Not mandatory, but it could be more maintainable, to transform to uppercase with CSS instead of this file.